### DISCUSSION TITLE
Is there any way to predict performance of ceph cluster?

### DISCUSSION THREAD

**Discussion by zhucan** [rook/rook#11025] [Q&A]
1. No network bottlenecks
2. No cpu and mem bottlenecks
3. Know the iops of the raw with with bs=4k
4. Know the total nums  of osd
5. Know the total nums of node

we want to predict performance of the ceph cluster with block volume and rgw.

**Comment by travisn** [comment]
You will need to test a specific environment to predict the performance in similar clusters. There are too many variables to answer in general.

**Reply by zhucan** [reply]
Is there a reference document?

**Reply by travisn** [reply]
You can find some references published online, for example I see [this one](https://access.redhat.com/documentation/en-us/red_hat_ceph_storage/4/pdf/hardware_guide/red_hat_ceph_storage-4-hardware_guide-en-us.pdf), though it is not specific to Rook deployments.

**Reply by zhucan** [reply]
Thanks, I will take a look.
